core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
-            "required": false,
-            "schema": {
-              "type": "array",
-              "items": {
-                "type": "string"
-              }
+    "openapi": "3.0.3",
+    "info": {
+        "title": "xRegistry API",
+        "description": "xRegistry API. See spec sections (e.g. 'Registry Capabilities', 'Retrieving the Registry Model', 'Doc', etc.) for details.",
+        "version": "1.0-RC1"
+    },
+    "servers": [],
+    "security": [
+        {
+            "api_key": []
+        }
+    ],
+    "paths": {
+        "/": {
+            "get": {
+                "operationId": "getRootDocument",
+                "tags": ["Root"],
+                "description": "Gets the root registry document. Query parameters 'inline', 'filter', 'doc', and 'specversion' modify the view. (See spec: Retrieving the Registry, Inline, Filter, and Doc)",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/inline"
+                    },
+                    {
+                        "$ref": "#/components/parameters/filter"
+                    },
+                    {
+                        "$ref": "#/components/parameters/doc"
+                    },
+                    {
+                        "$ref": "#/components/parameters/specversion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The root registry document.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/RegistryEntity"
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/BadRequest"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/Unauthorized"
+                    },
+                    "500": {
+                        "$ref": "#/components/responses/InternalServerError"
+                    }
+                }
+            },
+            "put": {
+                "operationId": "upsertDocument",
+                "tags": ["Root"],
+                "description": "Uploads a registry document and upserts its contents into the registry. (See spec: Creating or Updating Entities, Registry Entity)",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/inline"
+                    },
+                    {
+                        "$ref": "#/components/parameters/filter"
+                    },
+                    {
+                        "$ref": "#/components/parameters/doc"
+                    },
+                    {
+                        "$ref": "#/components/parameters/specversion"
```

Comment:
Missing: epoch, nodefaultversionid, nodefaultversionsticky, noepoch, noreadonly
that are missing on write operations.
At the resource/version level there's also "setdefaultversionid"